compiler/crates/relay-lsp/src/server/lsp_state.rs

```diff
@@ -422,17 +428,21 @@ impl<TPerfLogger: PerfLogger + 'static, TSchemaDocumentation: SchemaDocumentatio
             1,
         )?;
 
-        let info = match feature {
-            Feature::GraphQLDocument(executable_document) => FeatureResolutionInfo::GraphqlNode(
-                create_node_resolution_info(executable_document, position_span)?,
-            ),
-            Feature::DocblockIr(docblock_ir) => FeatureResolutionInfo::DocblockNode(DocblockNode {
-                resolution_info: create_docblock_resolution_info(&docblock_ir, position_span)
-                    .ok_or(LSPRuntimeError::ExpectedError)?,
-                ir: docblock_ir,
-            }),
-        };
-        Ok(info)
+        match feature {
+            Feature::ExecutableDocument(executable_document) => {
+                Ok(FeatureResolutionInfo::GraphqlNode(
+                    create_node_resolution_info(executable_document, position_span)?,
+                ))
+            }
+            Feature::DocblockIr(docblock_ir) => {
+                Ok(FeatureResolutionInfo::DocblockNode(DocblockNode {
+                    resolution_info: create_docblock_resolution_info(&docblock_ir, position_span)
+                        .ok_or(LSPRuntimeError::ExpectedError)?,
+                    ir: docblock_ir,
+                }))
+            }
+            Feature::SchemaDocument(_) => Err(LSPRuntimeError::ExpectedError),
```

Comment:
This is just temporary. https://github.com/facebook/relay/pull/4670 adds proper support.